### PR TITLE
property owner now supports arrays

### PR DESCRIPTION
**Pull request by ArnoldSmith86**
You can now add multiple player names to `owner`. Should be very interesting with the new COMPUTE function.

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: http://212.47.248.129:3043/pr-test (or any other room on that server)

**Comment by RaphaelAlvez** [comment]
Proper team hands?

**Comment by ArnoldSmith86** [comment]
Well, not _proper_.. Because there's no way yet to set the `owner` to an array `onEnter` (well, not to the current "team").

You can have _proper_ team hands right now if all team members set the same name though.

But with this you could make a complete holder visible to one team.. And with witchcraft you can of course make different holders at the same position visible to different teams/players.

**Comment by RaphaelAlvez** [comment]
@ArnoldSmith86 maybe you should add teams on top of this.
Can you provide a use case or reason for the current state of this PR?

**Review by RaphaelAlvez — CHANGES_REQUESTED** [review]
The code works.
Tested on 3 windows
image

My feedback would be to make this compatible with teams.
This is just feedback and you can merge if you want.

File: client/js/widgets/widget.js [hunk 1]
```diff
@@ -364,11 +370,7 @@ class Widget extends StateManaged {
   }
 
   updateOwner(oldName, newName) {
-    const o = this.p('owner');
-    if(o && o == oldName)
-      this.domElement.classList.add('foreign');
-    if(o && o == newName)
-      this.domElement.classList.remove('foreign');
+    this.domElement.className = this.classes();
```
Comment (UltimateGeek) [inline client/js/widgets/widget.js:373]: This no longer uses `oldName` nor `newName` parameters. Remove them? Rename it to `updateClasses()`?

Comment (ArnoldSmith86) [reply client/js/widgets/widget.js:373]: I don't know, I feel like the caller of that function doesn't know it's handled with classes. The caller wants to tell the widget that the current player changed its name.

**Review by RaphaelAlvez — APPROVED** [review]
Last reviews has been addressed. Approved

File: client/js/widgets/widget.js (hunk 1 above)
Comment (UltimateGeek) [reply client/js/widgets/widget.js:373]: It seems misleading to have parameters that are unused. Any typical static analysis tool and IDE would flag this.

Comment (ArnoldSmith86) [reply client/js/widgets/widget.js]: You're right. I removed the parameters. Renaming the function still seems weird to me.

**Review by UltimateGeek — APPROVED** [review]
Thank you for the update! :-)

**Review by Casbuild — APPROVED** [review]

**Pull request merged**